apps/web/src/app/teams/[id]/alerts/settings/slow-review/page.tsx

```
@@ -0,0 +1,93 @@
+            <Divider />
+
+            {!integration && (
+              <Stack p="md">
+                <AlertEnableSlack />
+              </Stack>
+            )}
+
+            {integration && (
+              <>
+                <AlertBaseFields form={form} />
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)